submissions/krylenger/friends/app.js

```diff
@@ -221,6 +224,11 @@ const addEventListeners = (usersData) => {
       filtersState.gender,
       filtersState.countries = null;
     search.value = "";
+    asideFilterAge.value = null;
+    asideRangeAgeOutput.textContent = '0 --> 120';
+    asideRadioGendersAll.checked = true;
+    asideRadioGenderMale.checked, 
+      asideRadioGenderFemale.checked = false;
```

Comment:
put your input fields into form and use `reset` method

Reply:
a simple html thing helped me to remove several JS lines
thanks!
✅ Done